Description: Additions/common/VBoxVideo: mistyped name caused restoring the
 graphics mode to fail in the X.Org Additions
Origin: upstream, https://www.virtualbox.org/changeset/38929

--- trunk/src/VBox/Additions/common/VBoxVideo/Modesetting.cpp (revision 38923)
+++ trunk/src/VBox/Additions/common/VBoxVideo/Modesetting.cpp (revision 38929)
@@ -178,5 +178,5 @@
 
     VBoxVideoCmnPortWriteUshort(VBE_DISPI_IOPORT_INDEX,
-                                VBE_DISPI_ENABLED);
+                                VBE_DISPI_INDEX_ENABLE);
     fFlags = VBoxVideoCmnPortReadUshort(VBE_DISPI_IOPORT_DATA);
     if (pcWidth)
